redaxo/src/core/lib/sql/sql.php

```diff
@@ -70,6 +70,9 @@ class rex_sql implements Iterator
     /** @var int */
     protected $DBID; // ID der Verbindung
 
+    /** @var string */
```

Comment:
```suggestion
    /**
      * Store the lastInsertId per rex_sql object, so rex_sql objects don't override each other because of the shared static PDO instance.
      *
      * @var string
      */
```